pkg/apis/core/validation/shoot.go

```diff
@@ -865,6 +870,16 @@ func validateKubernetes(kubernetes core.Kubernetes, dockerConfigured, shootHasDe
 			}
 		}
 
+		if kubeAPIServer.AccessControl != nil {
+			allErrs = append(allErrs, validateAccessControlAction(kubeAPIServer.AccessControl.Action, fldPath.Child("kubeAPIServer", "accessControl"))...)
+
+			if kubeAPIServer.AccessControl.Source.IPBlocks == nil {
+				allErrs = append(allErrs, field.Required(fldPath.Child("kubeAPIServer", "accessControl", "source", "ipBlocks"), "is required when access control action provided"))
```

Comment:
I don't get it - why is this marked as optional in the API when we require it here? Remove the "optional" tags from the API?